src/bse.py

```diff
@@ -0,0 +1,38 @@
+import requests
+from bs4 import BeautifulSoup
+class Bse:
+      def __init__(self,symbol):
+            googleUrl=f'https://www.google.co.in/search?q=bse+{symbol}+stock+price'
```

Comment:
Why are making use of google?

Reply:
it's difficult to search on BSE site because it also requires company symbol . So that's why I used google